src/main/resources/creatures.json

```diff
@@ -311,7 +311,7 @@
       "tags": [
         "CORPSE"
       ],
-      "drop": [
+      "drops": [
```

Comment:
@mafagafogigante, as I remember `drops` validation was added by your own. Anyway, very good that it helps with validation. *thumb limb*